### ISSUE TITLE
Python Require on Windows7 when installing

### ISSUE THREAD

**Issue by educatedrisk** [chjj/tty.js#116]
Error: Can't find Python executable "python", you can set the PYTHON env variable.

**Comment by daniellucas** [comment]
Yes, I got the same problem installing on Windows8, definitely the documentation should mark it as required.

P.S.: I have installed the Python 3.x.x, but the version should be 2.5.x ~ 2.9.x
